### PR TITLE
moved business logic and db interactions into CoffeeShopDbService. Ad…

### PR DESCRIPTION
**Pull request by DiskReader**
…ded connction string

### CODE REVIEW

File: CoffeeShop/Services/CoffeeShopDbService.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
+﻿using CoffeeShop.Context;
+using CoffeeShop.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace CoffeeShop.Services
+{
+    public class CoffeeShopDbService : ICoffeeShopDbService
```
Comment (pavelGagarin0870) [inline CoffeeShop/Services/CoffeeShopDbService.cs]: Remove db from name

File: CoffeeShop/Interfaces/ICoffeeShopDbService.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿namespace CoffeeShop.Interfaces
+{
+    public interface ICoffeeShopDbService
```
Comment (pavelGagarin0870) [inline CoffeeShop/Interfaces/ICoffeeShopDbService.cs]: Remove db from name

File: CoffeeShop/Controllers/CoffeeController.cs [hunk 3]
```diff
@@ -9,87 +8,44 @@ namespace CoffeeShop.Controllers
...
     {
         private readonly ILogger<CoffeeController> _logger;
+        private readonly ICoffeeShopDbService _service;
 
-        private CoffeeShopContext db = new CoffeeShopContext();
-
-        public CoffeeController(ILogger<CoffeeController> logger)
+        public CoffeeController(ILogger<CoffeeController> logger, ICoffeeShopDbService service)
         {
             _logger = logger;
```
Comment (pavelGagarin0870) [inline CoffeeShop/Controllers/CoffeeController.cs]: you inject logger, but never use it

File: CoffeeShop/Interfaces/ICoffeeShopDbService.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿namespace CoffeeShop.Interfaces
```
Comment (pavelGagarin0870) [inline CoffeeShop/Interfaces/ICoffeeShopDbService.cs]: Create folder Services

**Review by CrazyKamenwik — APPROVED** [review]

**Review by pavelGagarin0870 — APPROVED** [review]

**Pull request merged**